Tests/SwiftDocCTests/Rendering/RenderNodeTranslatorTests.swift

```diff
@@ -58,6 +58,27 @@ class RenderNodeTranslatorTests: XCTestCase {
         return paragraph
     }
     
+    private func getRenderNodeArticleFromBundlePath(
```

Comment:
A few notes about this test helper:
- It's only used I one test.
- In that test it's called multiple times which load and process the entire "Test Bundle" each time.
- Swift doesn't use a `get` prefix naming convention

If you move this test helper into the test and move the test bundle loading out of the helper it would be able to load "Test Bundle" just once (with all the extra articles added in one go) and then the helper could be used to only read an article from that context.